### ISSUE TITLE
Vanilla merge - forums merge error

### ISSUE THREAD

**Issue by euantorano** [mybb/merge-system#266]
While debugging #264, I ran into the following error:

```
SQL Error:
1366 - Incorrect integer value: '' for column `mybb`.`mybb_forums`.`pid` at row 1
Query:
UPDATE mybb_forums SET `pid`='', `parentlist`='37' WHERE fid='37' LIMIT 1
```

**Comment by euantorano** [comment]
Note: I got around this by adding the following chunk of code [here](https://github.com/mybb/merge-system/blob/feature/boards/vanilla/forums.php#L115):

```php
			if (empty($forum['updatefid'])) {
				continue;
			}
```

**Comment by luisalvarado** [comment]
Another one, you are on fire. How can I test this

**Comment by euantorano** [comment]
> Another one, you are on fire. How can I test this

I need to open another PR with the above code modification in, which I'll do along with a fix for #268 this week.
